allauth/usersessions/signals.py

```diff
@@ -1,8 +1,14 @@
+from django.dispatch import Signal
+
 from allauth.account import app_settings
 
 from .models import UserSession
 
 
+user_agent_changed = Signal()
```

Comment:
If the goal is to create an audit trail, it would be good to pass `from_user_agent` (=the old one) and `to_user_agent`) as arguments to this signal.